content/international-returners.md

```diff
@@ -2,12 +2,10 @@
 title: Return to teaching from overseas
 image: /assets/images/international-returners-dt.jpg
 backlink: "../../"
-#right_column:
-#  ctas:
-#    - title: Find out more at our next online event.
-#      link_text: "Sign up"
-#      link_target: "https://www.eventbrite.co.uk/e/return-to-teach-in-england-information-for-uk-teachers-working-overseas-tickets-141969770405"
-#      icon: "icon-calendar"
+description: |-
+  We want to help you return to teach in England. We provide courses, webinars
```

Comment:
Trying to get away from the "we" and put more emphasis on the "you". Something like:

"Return to England to teach with the help of a teacher training adviser. Courses and events are available to help make your transition easier."